lib/aca_entities/serializers/xml/fdsh/pvc/medicare/operations/medicare_request_to_xml.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module Serializers
+    module Xml
+      module Fdsh
+        module Pvc
+          module Medicare
+            module Operations
+              # RRV Request to payload xml
+              class MedicareRequestToXml
+                send(:include, Dry::Monads[:result, :do, :try])
+
+                # Transforms a primary request object to a request XML.
+                # @param [::AcaEntities::Fdsh::Ssa::H3::SsaCompositeRequest] the primary request
+                def call(medicare_request)
+                  request_xml = yield convert_to_xml(medicare_request)
+                  _validation_result = yield validate_xml(request_xml)
+                  Success(request_xml)
+                end
+
+                def convert_to_xml(non_esi_request)
+                  encode_result = Try do
+                    ::AcaEntities::Serializers::Xml::Fdsh::Rrv::Medicare::EesDshBatchRequestData.domain_to_mapper(non_esi_request).to_xml
```

Comment:
this is still pointing to rrv namespace, are we reusing or missed this by mistake ?